app/generators/questionnaire_generator.rb

```diff
@@ -317,22 +323,32 @@ def checkbox_option_body(question_id, option)
       }
       tag_options = add_shows_hides_questions(tag_options, option[:shows_questions], option[:hides_questions])
       wrapped_tag = tag(:input, tag_options)
-      option_body_wrap(question_id, option[:title], option[:tooltip], wrapped_tag)
+      option_body_wrap(question_id, option, wrapped_tag, elem_id, 'true', response_id)
     end
 
-    def option_body_wrap(question_id, title, tooltip, wrapped_tag)
+    def option_body_wrap(question_id, option, wrapped_tag, answer_key, answer_value, response_id)
```

Comment:
is / kan answer_value hier iets anders zin dan `option[:title]`, zoniet, dan maar gewoon weghalen? (aangezien we nu toch de hele option hash meegeven).

(je hebt er vast een goede reden voor om ze te scheiden, maar toch)

Reply:
Ja deze option_body_wrap functie wordt zowel gebruikt door radios als door checkboxes. Bij radios is het idd option[:title], maar bij checkboxes is het altijd `true` (en dat is niet de options[:title], die zit in de naam van het element verwerkt omdat we die splitsen bij checkboxes).